go/k8s/manifests/ctrl.yaml

```diff
@@ -67,6 +67,52 @@ spec:
               value: "minio_root_user"
             - name: UNKEY_VAULT_S3_ACCESS_KEY_SECRET
               value: "minio_root_password"
+            # Build S3 Storage (from depot-credentials secret)
+            #kubectl create secret generic depot-credentials \
+            # --from-literal=token=xxxx \
+            # --from-literal=s3-url=xxx \
+            # --from-literal=s3-access-key-id=xxx \
+            # --from-literal=s3-access-key-secret=xxx \
+            #   --namespace=unkey
+            - name: UNKEY_BUILD_S3_URL
+              valueFrom:
+                secretKeyRef:
+                  name: depot-credentials
+                  key: s3-url
+            - name: UNKEY_BUILD_S3_EXTERNAL_URL
+              valueFrom:
+                secretKeyRef:
+                  name: depot-credentials
+                  key: s3-url
+            - name: UNKEY_BUILD_S3_BUCKET
+              value: "build-contexts"
+            - name: UNKEY_BUILD_S3_ACCESS_KEY_ID
+              valueFrom:
+                secretKeyRef:
+                  name: depot-credentials
+                  key: s3-access-key-id
+            - name: UNKEY_BUILD_S3_ACCESS_KEY_SECRET
+              valueFrom:
+                secretKeyRef:
+                  name: depot-credentials
+                  key: s3-access-key-secret
+            # Depot Configuration
+            #kubectl create secret docker-registry depot-registry \
+            # --docker-server=registry.depot.dev \
+            # --docker-username=x-token \
+            # --docker-password=xxx \
+            # --namespace=unkey
```

Comment:
can we put this into a readme so its easier to find since we will need this in prod too?